`promisifyhelper` should be `promisifyHelper`

At https://github.com/compulim/azure-storage-fs/blob/master/lib/AzureBlobFS.js#L15, `promisifyhelper` should be case sensitive.

```
Cannot find module './util/promisifyhelper' from 'AzureBlobFS.js'
      
  at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:191:17)
  at Object.<anonymous> (node_modules/azure-storage-fs/lib/AzureBlobFS.js:11:37)
```

--- a/lib/AzureBlobFS.js
+++ b/lib/AzureBlobFS.js
@@ -12,7 +12,7 @@ const Promise = require('bluebird');
 const readAll = require('./util/readAll');
 const whilst = require('./util/whilst');
 const { R_OK, W_OK, X_OK } = require('fs');
-const { promisifyObject, toCallback } = require('./util/promisifyhelper');
+const { promisifyObject, toCallback } = require('./util/promisifyHelper');
 
 const
   DEFAULT_OPTIONS = {

